frontend/src/pages/IstioConfigList/IstioConfigListPage.tsx

```diff
@@ -99,20 +101,42 @@ class IstioConfigListPageComponent extends FilterComponent.Component<
     );
 
     if (namespacesSelected.length !== 0) {
-      this.fetchConfigs(namespacesSelected, istioTypeFilters, istioNameFilters, configValidationFilters, activeToggles);
+      this.setState({ listItems: [] });
+      if (isMultiCluster()) {
```

Comment:
You shouldn't need this if/else since for single cluster the loop will only iterate once for the one cluster.

Reply:
Actually since it looks like the istio config is aggregated across all clusters on the backend, do you need to make a request per cluster?

Reply:
this is for the case when serverConfig.clusters is empty which was happening on testing environment

Reply:
the istio config is returned from backend per cluster (called per cache/client for cluster), it is not aggregated across all clusters, otherwise that would require a lot of API JSON changed

Reply:
Took me some time to see too this, now the handler calls the `GetIstioConfigListPerCluster` function and not the former one which is the one that is aggregated now.

I saw that the aggregated function, `GetIstioConfigList` is called by for example the `GetServiceList` service, which can include Istio configs. In that service, now the Istio configs is aggregated, and the service list is also aggregated. If we are appending Istio configs (in the GetIstioConfigList) with no cluster information, then a service from a cluster will have, let's say, destination rules for all of the clusters?.

Reply:
Good catch, I missed that, for GetServiceList it will call GetIstioConfigListPerCluster

Reply:
Sure, no problem, there are also another references to the list function, so I think in all places will be required to used the one with the cluster. 

Will be required the aggregated function?.

Reply:
The aggregated function GetIstioConfigList is left for those areas when multicluster is not applied yet or is in still progress and is out of scope from this PR.
For the rest of places where GetIstioConfigList is called we will be back later. Otherwise the scope of this PR becomes endless.

Reply:
there are several subtasks created to cleanup the code https://github.com/kiali/kiali/issues/5618